winston.js

```diff
@@ -11,6 +11,7 @@ const logger = winston.createLogger({
   ),
   defaultMeta: { service: `federalistapp-${config.app.app_env}` },
   transports: [new winston.transports.Console()],
+  silent: config.log.silent,
```

Comment:
silence logs during testing to improve speed and reduce noise. Uncaught exceptions are still logged.